argo/providers/vai/provider.go

```diff
@@ -228,7 +217,47 @@ func (vaip VAIProvider) CreateRun(ctx context.Context, providerConfig VAIProvide
 		Artifacts:         runDefinition.Artifacts,
 	}
 
-	return runId, vaip.SubmitRun(ctx, providerConfig, run)
+	pipelineClient, err := aiplatform.NewPipelineClient(ctx, option.WithEndpoint(providerConfig.vaiEndpoint()))
+	if err != nil {
+		return "", err
+	}
+	defer pipelineClient.Close()
+
+	parameters := make(map[string]*aiplatformpb.Value, len(run.RuntimeParameters))
+	for name, value := range run.RuntimeParameters {
+		parameters[name] = &aiplatformpb.Value{
+			Value: &aiplatformpb.Value_StringValue{
+				StringValue: value,
+			},
+		}
+	}
+
+	pipelineJob := &aiplatformpb.PipelineJob{
+		Labels:         run.Labels,
```

Comment:
I think the `VAIRun` created above is not necessary. I fact, I think we can remove that struct entirely.

Reply:
Agreed and have removed, only 2 values used from struct so passing as params now